smart-metastore/src/main/java/org/smartdata/metastore/utils/MetaStoreUtils.java

```diff
@@ -338,7 +340,12 @@ public static void initializeDataBase(
               + " dest varchar(4096) NOT NULL,\n"
               + " period bigint(20) NOT NULL\n"
               + ") ;",
-          "CREATE INDEX backup_file_rid_idx ON backup_file (rid);"
+          "CREATE INDEX backup_file_rid_idx ON backup_file (rid);",
+          "CREATE TABLE file_state (\n"
+              + " path varchar(1000) PRIMARY KEY,\n"
```

Comment:
Do the file_state will auto sync with the file_info? Like after add a file in hdfs, the file_state will add a row with this file, whose type is normal.

Reply:
No, file_state and file_info updated separately.